keyboards/splitkb/elora/rev1/rev1.h

```diff
@@ -0,0 +1,93 @@
+    MX103, MX107, MX111, MX115, MX119, MX123,        LPED3, RPED3,        MX223, MX219, MX215, MX211, MX207, MX203, \
+    MX104, MX108, MX112, MX116, MX120, MX124, MX131, MX130, MX230, MX231, MX224, MX220, MX216, MX212, MX208, MX204, \
+                         MX125, MX126, MX127, MX128, MX129, MX229, MX228, MX227, MX226, MX225                       \
+) \
+{ \
+    { MX131, LPED1, LPED2, LPED3, MX130, MX129, KC_NO, KC_NO }, \
+    { KC_NO, KC_NO, KC_NO, KC_NO, MX128, MX127, MX126, MX125 }, \
+    { MX118, MX117, MX121, MX122, MX123, MX124, MX120, MX119 }, \
+    { MX110, MX109, MX113, MX114, MX115, MX116, MX112, MX111 }, \
+    { MX102, MX101, MX105, MX106, MX107, MX108, MX104, MX103 }, \
+    { KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO }, \
+                                                                \
+    { KC_NO, KC_NO, MX229, MX230, RPED3, RPED2, RPED1, MX231 }, \
+    { MX225, MX226, MX227, MX228, KC_NO, KC_NO, KC_NO, KC_NO }, \
+    { MX219, MX220, MX224, MX223, MX222, MX221, MX217, MX218 }, \
+    { MX211, MX212, MX216, MX215, MX214, MX213, MX209, MX210 }, \
+    { MX203, MX204, MX208, MX207, MX206, MX205, MX201, MX202 }, \
+    { KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO }, \
+} //    A      B      C      D      E      F      G      H
+
+#define LAYOUT_myr( \
+    MX101, MX105, MX109, MX113, MX117, MX121,        LPED1, RPED1,        MX221, MX217, MX213, MX209, MX205, MX201, \
+    MX102, MX106, MX110, MX114, MX118, MX122,        LPED2, RPED2,        MX222, MX218, MX214, MX210, MX206, MX202, \
+    MX103, MX107, MX111, MX115, MX119, MX123,        LPED3, RPED3,        MX223, MX219, MX215, MX211, MX207, MX203, \
+    MX104, MX108, MX112, MX116, MX120, MX124, MX131, MX130, MX230, MX231, MX224, MX220, MX216, MX212, MX208, MX204, \
+                         MX125, MX126, MX127, MX128, MX129, MX229, MX228, MX227, MX226, MX225,                      \
+    MYRL1, MYRL2, MYRL3, MYRL4,        MYRL5,               MYRR1, MYRR2, MYRR3, MYRR4,        MYRR5                \
+) \
+{ \
+    { MX131, LPED1, LPED2, LPED3, MX130, MX129, KC_NO, KC_NO }, \
+    { KC_NO, KC_NO, KC_NO, KC_NO, MX128, MX127, MX126, MX125 }, \
+    { MX118, MX117, MX121, MX122, MX123, MX124, MX120, MX119 }, \
+    { MX110, MX109, MX113, MX114, MX115, MX116, MX112, MX111 }, \
+    { MX102, MX101, MX105, MX106, MX107, MX108, MX104, MX103 }, \
+    { MYRL1, MYRL2, MYRL3, MYRL4, MYRL5, KC_NO, KC_NO, KC_NO }, \
+                                                                \
+    { KC_NO, KC_NO, MX229, MX230, RPED3, RPED2, RPED1, MX231 }, \
+    { MX225, MX226, MX227, MX228, KC_NO, KC_NO, KC_NO, KC_NO }, \
+    { MX219, MX220, MX224, MX223, MX222, MX221, MX217, MX218 }, \
+    { MX211, MX212, MX216, MX215, MX214, MX213, MX209, MX210 }, \
+    { MX203, MX204, MX208, MX207, MX206, MX205, MX201, MX202 }, \
+    { MYRR1, MYRR2, MYRR3, MYRR4, MYRR5, KC_NO, KC_NO, KC_NO }, \
+} //    A      B      C      D      E      F      G      H
```

Comment:
Please moves these to info.json.   `qmk info -kb (name) -f json` can help with that.